src/Stratis.Bitcoin.IntegrationTests/Program.cs

```diff
@@ -2,8 +2,10 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
+using Stratis.Bitcoin.IntegrationTests.Miners;
```

Comment:
We should consider an SC integration test project (before or after the commit? I am  not sure)